src/v/redpanda/application.cc

```diff
@@ -2743,15 +2747,31 @@ void application::wire_up_and_start(::stop_signal& app_signal, bool test_mode) {
     } else if (auto id = _node_overrides.node_id(); id.has_value()) {
         vlog(
           _log.warn,
-          "Overriding node ID: {} -> {}",
+          "Overriding node ID: {} -> {} [forced?]",
           config::node().node_id(),
-          id);
+          id,
+          has_id && force_override);
```

Comment:
missing a fmt placeholder?

Reply:
weird...i recall typing it out. git snafu maybe.

Reply:
done